README.md

```diff
@@ -1 +1,79 @@
-# Capstone_projects
\ No newline at end of file
+# Capstone_projects
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Microverse Capstone Project
+
+ This project is the Final capstone project of my first module at microverse. It is aim at building responsive UI of an online website for  Music Entertainment Industry , so in this project I built the home and the about page of the conference page (and readable) on all screens.
+The template I used to create this template was originally designed by [Cindy Shin in Behance.](https://www.behance.net/adagio07)
+
+https://github.com/ademibowale/Capstone_projects
+https://www.loom.com/share/201a1c930c084c29a9f3cbf1198e59bb
```

Comment:
- [ ] Kindly remove the loom video here and put it in your PR message as a link.